### ISSUE TITLE
Let's get rid of Flash [Due: 09/2018]

### ISSUE THREAD

**Issue by jiri-janousek** [tiliado/nuvola-app-amazon-cloud-player#33]
Since this script is compatible with the Chromium backend, we could proceed further: Replace `"requirements": "Codec[MP3] Feature[Flash]"` with `"requirements": "Codec[MP3] Feature[MSE] Chromium[63]"`. For this, we need to check:

- [ ] Playback works with Flash plugin disabled (`Feature[Flash]` removed). I guess this will be okay presuming that Amazon detected functional MSE and didn't bother to use Flash plugin at all during your tests of the Chromium backend.
- [ ] There are no issues under Wayland, such as https://github.com/tiliado/nuvolaruntime/issues/372#issuecomment-355214224.

**Comment by a-m-s** [comment]
Sorry, flash is required. This is what I get with your suggested requirements:
![screenshot from 2018-03-11 23-02-04](https://user-images.githubusercontent.com/5088295/37259618-6d208d6e-2580-11e8-9f74-88f956a07760.png)
Without Flash you can only play music that you own; it won't play music "included with Prime".

**Comment by jiri-janousek** [comment]
Oh, that's a bummer. I've expected more from Amazon. Could you check whether Flash is also required in the genuine Chrome and then ask Amazon Cloud Player support whether there are any plans to get rid of Flash? You can say that Chrome makes more and more difficult to use Flash and [there is a roadmap for its removal](https://www.chromium.org/flash-roadmap).

Meantime, we could switch to Chromium, where Flash is more stable. Could you test following requirements under Wayland? The first line is a comment linking this issue for me not to forget Flash is still required.

```
"#requirements": "https://github.com/tiliado/nuvola-app-amazon-cloud-player/issues/33",
"requirements": "Codec[MP3] Feature[MSE] Feature[Flash]",
```

**Comment by a-m-s** [comment]
I get the same message in Chrome. I will send them a message.

**Comment by a-m-s** [comment]
I wrote an email to Amazon support. I got a standard reply ignoring my questions and telling me to clear my cookies or use a different browser.

Trying again, worded for idiots.

**Comment by a-m-s** [comment]
Still no luck. They keep telling me to call them. I don't fancy my chances of getting any operator that has a clue.

**Comment by jiri-janousek** [comment]
The Amazon support seems to be unbelievably incompetent :-(

**Comment by jiri-janousek** [comment]
Let's check the progress every 4 months - next check September 2018.

**Comment by a-m-s** [comment]
I believe this issue is now resolved too! :-D

**Comment by jiri-janousek** [comment]
Sure :-)

**Comment by jiri-janousek** [comment]
Released with **Nuvola 4.16.0**, more info: [Nuvola 4.16 Brings Libre.FM Scrobbling, Ivoox Service, and more](https://medium.com/nuvola-news/nuvola-4-16-86a089068784)
